### DISCUSSION TITLE
[AvaloniaNativeDragSource] Possibility Condition of while statement not valid?

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#11177] [Q&A] [answered]
Considering the following lines:
https://github.com/AvaloniaUI/Avalonia/blob/9f64bad365880fa12d29d17eaecd68b92504f35a/src/Avalonia.Native/AvaloniaNativeDragSource.cs#L21-L29

The while condition will always be false because Interactive->LayoutTable->Visual
https://github.com/AvaloniaUI/Avalonia/blob/9f64bad365880fa12d29d17eaecd68b92504f35a/src/Avalonia.Native/AvaloniaNativeDragSource.cs#L23-L24

The change occurred in PR #9553

Can I remove the while statement?

cc @grokys

**Comment by maxkatz6** [comment, marked as answer]
It looks like whole method can be removed with single TopLevel.GetTopLevel(visual as Interactive);
